client/my-sites/people/edit-team-member-form/edit-user-form.jsx

```diff
@@ -249,16 +256,27 @@ class EditUserForm extends React.Component {
 			return null;
 		}
 
+		const { translate, hasWPCOMAccountLinked, disabled, markChanged, isUpdating } = this.props;
+
 		return (
 			<form
 				className="edit-team-member-form__form" // eslint-disable-line
-				disabled={ this.props.disabled }
+				disabled={ disabled }
 				onSubmit={ this.updateUser }
-				onChange={ this.props.markChanged }
+				onChange={ markChanged }
 			>
-				{ editableFields.map( ( fieldId ) => this.renderField( fieldId, this.props.isUpdating ) ) }
+				{ editableFields.map( ( fieldId ) => this.renderField( fieldId, isUpdating ) ) }
+				{ hasWPCOMAccountLinked && (
+					<p className="edit-team-member-form__explanation">
+						{ translate(
+							'This user has a WordPress.com account, ' +
+								'they can change their First Name, Last Name, Display Name that ' +
```

Comment:
I'd update this to:

"This user has a WordPress.com account, only they are allowed to update their personal information through their WordPress.com profile settings"

I'd also not split this string up if possible so it's more searchable within the codebase.